tests/Integration/Integrations/MemcachedTest.php

```diff
@@ -26,10 +26,6 @@ protected function setUp()
     {
         parent::setUp();
 
-        if (PHP_MAJOR_VERSION < 7) {
```

Comment:
`Predis` integration tests (just recently merged to master), they were also skipped, so you may want to pull master and re-enabled them as well

Reply:
Ok will reenable them 🤞

Reply:
Phew, tests have passed as well 🕺